knowledge-portal/central/reverse_mapping.py

```diff
@@ -0,0 +1,89 @@
+# This script would take in a list of file names and find the reverse mappings
+# Assumptions:
+# manifests/<repo_name>.txt: Source repo name should be the manifest name
+# manifests have the convention: `source-->destination`
+import os
+import subprocess
+import sys
+import time
+from glob import glob
+
+
+def sp_run(s):
```

Comment:
You can remove this aswell as the commented lines in the end

Reply:
Ohh yeah sorry, will do that